spinalcordtoolbox/reports/assets/index.html

```diff
@@ -241,9 +246,19 @@
          for the case where the table should be empty (i.e. when search
          results correspond to nothing).
          */
+        b=document.createElement("div")
+        b.className = "fixed-table-toolbar";
+        b.style.paddingTop="10px";
+        b.innerHTML="<button class=\"btn btn-default\" type=\"button\" id=\"download_yaml_btn_274c\" onclick=\"download_yaml('❌')\" disabled>Download \"❌\"</button><button class=\"btn btn-default\" type=\"button\" id=\"download_yaml_btn_26a0\" onclick=\"download_yaml('⚠️')\" disabled>Download \"⚠️\"</button>"
```

Comment:
From https://github.com/neuropoly/spinalcordtoolbox/commit/d9163cc79135d23dfbbfffa08b23d8d9d9fc3be3, this line is what stands out to me. 

I'm not sure how `.innerHTML` is parsed, but I suspect it's handled differently (compared to our usage of the unicode characters elsewhere).

Reply:
Please disregard this comment -- I believe the problem is due to https://github.com/neuropoly/spinalcordtoolbox/pull/3253#issuecomment-822719480.